us_election_public_county_facts_dictionary.sql

```diff
@@ -0,0 +1,65 @@
+create table county_facts_dictionary
+(
+    column_name varchar(9) not null
+        constraint county_facts_dictionary_pk
+            primary key,
+    description varchar
+);
+
+alter table county_facts_dictionary
+    owner to postgres;
```

Comment:
To jest trochę niebezpieczne - setujesz ownera na usera, którego masz u siebie, a którego niekoniecznie muszą mieć wszyscy ustawionego u siebie tak samo. Pewnie mają, ale nigdy nie można tego zakładać.